components/Map/index.js

```diff
@@ -78,6 +78,7 @@ class MapContainer extends React.Component {
       `CREATE TABLE IF NOT EXISTS ${tableNameByYear} AS SELECT * FROM "${datasetFileName}"`; // query from parquet
 
     const startTime = performance.now(); // start the time tracker
+    setStartTime(startTime)
```

Comment:
@bphan002 can we rename `startTime` / `setStartTime` to a more informative name? E.g. pinLoadStartTime, so that we differentiate the way start time is used here vs the way it is used in `Map.jsx`.